### PR TITLE
UI Improvements to vertex drawing

### PR DESCRIPTION
**Pull request by TabLand**
1) Make the vertex fills grey and transparent for both circle & square shapes
   This makes it easier to see crease pattern error with large vertices
2) Vertex fill colours made consistent between circle and square shapes 
3) Add border to square shaped vertex, again for consistency

Fixes issue #324 

Screenshots to follow shortly...

### CODE REVIEW

**Comment by TabLand** [comment]
Screenshot from 2023-06-14 10-04-27
Screenshot from 2023-06-14 10-03-58
Screenshot from 2023-06-14 10-04-13
Screenshot from 2023-06-14 10-04-04

**Comment by TabLand** [comment]
Hey @Minh-Than,
What are your initial thoughts on this change just from reviewing the screenshots?

**Comment by Minh-Than** [comment]
I mean it certainly helps I think, but I don't really go past having the lowest sized vertex (no vertex) at all so I can't really comment much on that in sdetail.

@undertrox @qurben what do you guys think?

File: oriedita-data/src/main/java/oriedita/editor/Colors.java
Comment (maks2001916) [inline oriedita-data/src/main/java/oriedita/editor/Colors.java:43]: Hello, I'm a beginner developer, I'm interested in the oriedita project, I would like to take part in its development. As a developer, I don’t know much yet, but I would like to understand how the project works and works.

Sorry for posting here, I couldn't find any other way to get in touch.

Comment (TabLand) [reply oriedita-data/src/main/java/oriedita/editor/Colors.java:43]: Hi @maks2001916,

There's actually a small and hopefully easy issue in oriedita that I hadn't gotten around to logging yet. 
Happy to support you in the fault-find and patch preparation.

Comment (maks2001916) [reply oriedita-data/src/main/java/oriedita/editor/Colors.java:43]: Tell me about problem solving.

I want to warn that there may be some misunderstandings, since I do not know English and I translate messages. I hope there will be a minimum.

Comment (TabLand) [reply oriedita-data/src/main/java/oriedita/editor/Colors.java:43]: Please see #329

Comment (TabLand) [reply oriedita-data/src/main/java/oriedita/editor/Colors.java:43]: Hi @maks2001916,

I'll probably patch that above issue in the next week as it seems you're busy; however there are some other issues in this project we could potentially work on together (e.g SVG import). 

Please reach out to me via the email listed in my GitHub profile.

**Review by undertrox — APPROVED** [review]
sorry for the long delay, but i quite like the changes. Since it's been a while, i'll leave this open for a few more days in case anyone else wants to chime in too, but then I'd merge it if thats ok

**Comment by TabLand** [comment]
The only further addition I can think of is adding a toggle option in the preferences section, in case someone doesn't want this kind of vertex drawing?

Would that over complicate things?

**Comment by Minh-Than** [comment]
Don't think it will complicate things but I'm currently working on the preference window at the moment and I'm kind of bad at dealing with conflicts lol.

**Comment by TabLand** [comment]
No worries, happy to hold off on that until / unless it's requested

**Comment by undertrox** [comment]
Alright I'm gonna merge this then, once the settings window improvements are done, you can make a new pr for adding it to the settings

**Pull request merged**
